tcms/testruns/tests/test_admin.py

```diff
@@ -47,3 +48,23 @@ def test_superuser_can_delete_testrun(self):
                                             args=[self.test_run.pk]), {'post': 'yes'}, follow=True)
         self.assertRedirects(response, reverse('admin:testruns_testrun_changelist'))
         self.assertEqual(TestRun.objects.filter(pk=self.test_run.pk).exists(), False)
+
+
+class TestTestExecutionStatusAdmin(LoggedInTestCase):
+
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+        cls.superuser = UserFactory()
+        cls.superuser.is_superuser = True
+        cls.superuser.set_password('password')
+        cls.superuser.save()
+
+    def test_admin_testrun_execution_status_color(self):
+        self.client.login(  # nosec:B106:hardcoded_password_funcarg
+            username=self.superuser.username,
+            password='password')
```

Comment:
LoggedInTestCase makes sure that the user is already logged in so this shouldn't be necessary. Also this is accessible to non-superusers too.